services/web/server/src/simcore_service_webserver/projects/projects_handlers.py

```diff
@@ -10,6 +10,7 @@
 import aioredlock
 from aiohttp import web
 from jsonschema import ValidationError
+from models_library.projects import ProjectType
```

Comment:
this is probably a bad merge. can you revert it please? I think this will cause some issues.

Reply:
yep, thanks